test/cucumber/evaluation.feature

```diff
@@ -0,0 +1,29 @@
+Feature: vizualize evaluation
+	 As a teacher
+	 I want to see evaluation
+	 So I can see any details that i want
```

Comment:
# not a valuable motivation. Better is "So I can choose a evaluation that fits to the class"